### ISSUE TITLE
Has documentation been blocked for Africa?

### ISSUE THREAD

**Issue by gitn00b1337** [invertase/react-native-firebase#7822]
## Documentation Feedback

I am unable to reach the documentation. All other websites I use are working fine, and I used an online website status tool to check your website is up, which it said it was. I was wondering if your hosting provider has blocked Africa (or Tanzania) from access, as it's quite common to happen.

I can also access it with proxysite.com

**Comment by rasheedefficion** [comment]
Same here. Cant access the docs. What is heppening??

**Comment by mikehardy** [comment]
Unknown, having been in "Africa" though, I'm always quick to point out to others that it's a really big continent and (for instance) things going on in Morocco probably don't have much to do with something in Botswana :-). So being specific about countries would help

That said, our documentation site is hosted by Vercel, so more investigation here would involve asking vercel if they have any content blocks up for your specific location?

In the meantime all our docs are accessible on github as they are built from it, you may hopefully access them here: https://github.com/invertase/react-native-firebase/blob/main/docs/index.md if a proxy isn't working well for whatever reason

**Comment by gitn00b1337** [comment]
Correct, yet the entire contintent does in fact generally get blocked from providers (although SA often gets let through). But i did also state Tanzania, which for clarity is where I'm trying to access from

**Comment by mikehardy** [comment]
@gitn00b1337 indeed you specified well, my comment on location details was intended mostly for @rasheedefficion who provided only contintent-level info

Surprising that the continent as a whole is frequently blocked, on the surface that seems pretty ridiculous to the point of being shameful. New information to me.

That said I don't think this is actionable here - we use Vercel so blocking inquiries will have to go there / be resolved there, and today I checked this again using NordVPN which allows me to pick a point in Nairobi (as close to Tanzania as I can get from a network perspective) and rnfirebase.io loaded fine for me.

So I'm going to close this as at least for me not reproducible - perhaps it resolved for you as well? I hope so - if not reach out to Vercel support, and there's a backup workaround I provided above in the form of link to the git source for docs. Hopefully between those 3 avenues you're not stuck on your project. Cheers
